websites/S/Stan/presence.ts

```diff
@@ -50,59 +54,85 @@ const presence = new Presence({
     };
   } = {
     presence: {},
+    coverUrls: {},
+    strings: null,
     meta: {},
     startedSince: ~~(Date.now() / 1000),
```

Comment:
I know this PR didn't touch this line but using the bitwise NOT operator yields a slightly misleading value as opposed to Math.round (1 unit smaller)
```suggestion
    startedSince: Math.round(Date.now() / 1000),
```